### PR TITLE
[Inventory Template] - Updates to the inventory page

### LINKED ISSUE #116: [Inventory Template] - Updates to the inventory page
Updating for performance, navbar etc.

### PR DESCRIPTION
**Pull request by bhellema**
Fix #116 

Test URLs:
- Before: https://main--hubblehomes-com--aemsites.hlx.page/new-homes/idaho/boise-metro/nampa/adams-ridge/maple-loft/10418-rockaway-ridge-st/999
- After: https://issue-116--hubblehomes-com--aemsites.hlx.page/new-homes/idaho/boise-metro/nampa/adams-ridge/maple-loft/10418-rockaway-ridge-st/999

### CODE REVIEW

File: templates/blocks/available-at-locations/available-at-locations.css [hunk 1]
```diff
@@ -1,6 +1,9 @@
-.available-at-locations a {
-  font-weight: 400;
-  text-decoration: underline;
-  text-decoration-color: #afafaf;
-  color: var(--link-color);
-}
\ No newline at end of file
+.available-at-locations {
+  a {
```
Comment (codevoyage) [inline templates/blocks/available-at-locations/available-at-locations.css:2]: Just curious, is there a reason why this notation is preferred over .available-at-locations a?

**Comment by codevoyage** [comment]
The floorplan buttons look different from the original, not just in colour but also in height, width, and possibly radius.

The original has a different design for the floorplans on mobile. If sticking to the tabs on mobile, the spacing of the buttons need to be fixed.

The floorplan image in the tab is not centred. 

The floorplan tab colour looks different.

File: templates/inventory/inventory.js [hunk 2]
```diff
@@ -25,22 +37,18 @@ async function fetchRequiredPageData() {
 
 async function createPriceCell(homeDetails) {
   const { price } = homeDetails;
-  const numericPrice = price ? parseFloat(price.replace(/[^\d.-]/g, '')) : null;
-
```
Comment (codevoyage) [inline templates/inventory/inventory.js:29]: This was required to get only the numbers from the price, removing any special characters like commas or symbols.

Comment (bhellema) [reply templates/inventory/inventory.js:29]: No need for this as the data coming in is raw data without the formatting.

File: templates/blocks/available-at-locations/available-at-locations.css (hunk 1 above)
Comment (bhellema) [reply templates/blocks/available-at-locations/available-at-locations.css:2]: It's easier to read and maintaining css - https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_nesting/Using_CSS_nesting

**Comment by bhellema** [comment]
> The floorplan buttons look different from the original, not just in colour but also in height, width, and possibly radius.
I have had a conversation with Nate about the buttons in the slack channel and he agreed that a single color is good. https://adobe-dx-support.slack.com/archives/C06P5NMBH6W/p1718903803527419

> The original has a different design for the floorplans on mobile. If sticking to the tabs on mobile, the spacing of the buttons need to be fixed.  
I'm going to demo the change that I have, otherwise we need to include an accordion component for this as well which just adds to the dom size.

The floorplan image in the tab is not centred.
> fixed

The floorplan tab colour looks different.
> should be right now

**Comment by codevoyage** [comment]
I noticed that the “Request a Tour” button has been removed. Will this be consistent across all model and community pages as well?

**Review by codevoyage — APPROVED** [review]

**Pull request merged**